docs/locale/fr/LC_MESSAGES/userguide/recovery_device.po

```diff
@@ -0,0 +1,207 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) 2016-present, Scille SAS
+# This file is distributed under the same license as the Parsec package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, 2023.
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: Parsec 2.16.0-a.0+dev\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2023-11-10 09:43+0000\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language: fr\n"
+"Language-Team: fr <LL@li.org>\n"
+"Plural-Forms: nplurals=2; plural=(n > 1);\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=utf-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Generated-By: Babel 2.13.0\n"
+
+#: ../../userguide/recovery_device.rst:7
+msgid "Recover devices"
+msgstr "Récupérer des appareils"
+
+#: ../../userguide/recovery_device.rst:9
+msgid ""
+"A *recovery device* is a backup of an existing device. If you lose access to "
+"Parsec (you forget the password), the recovery device will allow you to "
+"recreate a new device and set a new password."
+msgstr ""
+"Un *appareil de récupération* est une sauvegarde d'un appareil existant. Si "
+"vous perdez accès à Parsec (vous avez oublié votre mots de passe), "
```

Comment:
```suggestion
"vous perdez l'accès à un appareil Parsec (vous avez oublié votre mots de passe), "
```

Reply:
The original 

> If you lose access to Parsec

Does not translate to that

Reply:
Is true that we lost access "to a device" and not necessary "to Parsec". Maybe we can translate here, and change the original later?

Reply:
done